TestFoundation/TestNSString.swift

```diff
@@ -77,7 +77,6 @@ class TestNSString : XCTestCase {
             ("test_initializeWithFormat", test_initializeWithFormat),
             ("test_initializeWithFormat2", test_initializeWithFormat2),
             ("test_initializeWithFormat3", test_initializeWithFormat3),
-            ("test_deletingLastPathComponent", test_deletingLastPathComponent),
```

Comment:
Since this is just moving the contents rather than changing, please leave it here. If necessary, you can add the `test_appendingPathComponent` above the `test_deletingLastPathComponent` to minimise the diffs.